spec/controllers/spree/api_keys_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Spree::ApiKeysController, type: :controller, performance: true do
+  routes { Spree::Core::Engine.routes }
+
+  include AuthenticationHelper
+  include ControllerRequestsHelper
+
+  let(:user) { create(:user) }
+  let(:redirect_path) { "#{spree.account_path}#/developer_settings" }
+
+  before do
+    allow(controller).to receive(:spree_current_user) { user }
+  end
+
+  describe "create" do
+    it "creates a new api key" do
+      spree_api_key = user.spree_api_key
+      spree_post :create
+
+      expect(user.reload.spree_api_key).to_not eq(spree_api_key)
```

Comment:
Check this out:
```suggestion
      expect { spree_post :create }.to change { user.reload.spree_api_key }
```

Reply:
Cool! I am not so familiar with rspec. Usually I use minitest but I am glad to learn some rspec magic!